src/pulse3D/utils.py

```diff
@@ -13,12 +14,55 @@
 from .constants import CONTRACTION_TIME_UUID
 from .constants import RELAXATION_TIME_UUID
 from .constants import TIME_DIFFERENCE_UUID
+from .constants import TWENTY_FOUR_WELL_PLATE
+from .constants import WELL_INDEX_UUID
 from .constants import WIDTH_FALLING_COORDS_UUID
 from .constants import WIDTH_RISING_COORDS_UUID
 from .constants import WIDTH_UUID
 
 logger = logging.getLogger(__name__)
 
+def get_start_and_end_times(plate_recording, 
+                            start_time: Union[int, float, Dict[str, Union[int, float]]]=0, 
+                            end_time: Union[int, float, Dict[str, Union[int, float]]]=np.inf):
+    """[summary]
+
+    Args:
+        plate_recording ([type]): [description]
+        start_time ([type]): [description]
+        end_time ([type]): [description]
+    """
+
+    assert isinstance(start_time, (float, int)), 'Windowing only supports plate-specific start times -- provide a float or int value.'
+    assert isinstance(end_time, (float, int)), 'Windowing only supports plate-specific start times -- provide a float or int value.'
```

Comment:
I think we'll need to return an error here, or raise an exception with a meaningful error string instead of an assert, since we can't handle an assert at runtime

Reply:
I resolved this, since we're only doing plate-specific start/end times.